views: add accessibilityDataSensitive to harden against pwd exfiltration

researchxxl opened the pull request:
reason https://android-developers.googleblog.com/2025/12/enhancing-android-security-stop-malware.html

Catfriend1 commented:
ToDo:
- Close SettingsActivity after sharedPref import (or refresh it)
- Use a handler to restart syncthing in the background (hangs when enabling fs debug facility)
- Respect run condition when importing backup while run conditions do not apply (and syncthing is initially in state_disabled)
- Add Icon on drawer

Catfriend1 commented:
Testing completed at commit https://github.com/Catfriend1/syncthing-android/pull/61/commits/f0b2cc07d47289007f2b61097e9c986206e84b34 .
@Review/Changes-Approved.
lgtm.

researchxxl committed:
views: add accessibilityDataSensitive

https://android-developers.googleblog.com/2025/12/enhancing-android-security-stop-malware.html
